test/beer-router-test.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
```

Comment:
Great job volunteering yourself for code review. It was fun debugging your tests and I hope you understand things better.